--- a/Makefile
+++ b/Makefile
@@ -36,10 +36,8 @@
 CR_GOAL = $(NAME).exe
 WINCC = $(RAW_CC) $(RAW_FLAGS) -D__WIN32
 
-# Generic "how to compile C files"
-CC = $(RAW_CC) $(RAW_FLAGS) -D__UNIX
-.c.o:   
-	$(CC) -c $<
+%.o : %.c
+	$(CC) $(CFLAGS) $(CPPFLAGS) $(RAW_FLAGS) -D__UNIX -c $<
 
 
 # Definitions we'll need later (and that should rarely change)
@@ -115,7 +113,7 @@
 #---------------------------------------------------------------------
 
 foremost: $(OBJ)
-	$(CC) $(OBJ) -o $(NAME) $(LINK_OPT)
+	$(CC) $(OBJ) -o $(NAME) $(LDFLAGS)
 
 
 #---------------------------------------------------------------------
